### PR TITLE
Pinapple salad hair fluff

### PR DESCRIPTION
**Pull request by Aurorablade**
ckey is Pineapple Salad
Character: Dan Jello
Fluff type: Hair

had to revist the one i did for zeke for this and i did a recolor of the bottle just to make sure we had a item to use for this.

:cl:
rscadd:Pineapple salad hair fluff
/:cl:

### CODE REVIEW

**Comment by variableundefined** [comment]
Travis y

**Comment by variableundefined** [comment]
I think a new folder / new dmi file should be created for fluff hair.

Or it should be attached to a slime people .dmi file and follow the convention I used when I refactored the DMI, I don't want hair to become a total mess again.

**Comment by Aurorablade** [comment]
@variableundefined thats the thing..it is...Zekes fluff hair is under skrell. The thing about slime people hair is its technically human hair with a trasnperency effect. I mean i can certainly make a folder for fluff hair. but its still following the the rules you got, just slime hair and human hair are the same thing, I just have this code restricted.

File: code/modules/mob/new_player/sprite_accessories/human/human_hair.dm [hunk 1]
```diff
@@ -682,4 +682,10 @@
 
 ///////////////////////////////////
 //////END POLARIS HAIRSTYLES///////
-//////////////////////////////////
\ No newline at end of file
+//////////////////////////////////
+
+///Fluff HairStyles//
+datum/sprite_accessory/hair/fluff/pinapple_fluff_hair //Pineapple Salad hair fluff its for a slime..has to go under human
```
Comment (variableundefined) [inline code/modules/mob/new_player/sprite_accessories/human/human_hair.dm]: missing a /

**Pull request merged**